src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect, useState } from 'react'
+import { getArticles } from '../../api'
+import ArticleCard from './ArticleCard'
+
+export default function ArticlesList() {
+    const [articlesListItems, setArticlesListItems] = useState([])
+
+    useEffect(()=>{
+        getArticles().then(({articles, total_count})=>{
+            setArticlesListItems(articles)
+        })
+    }, [])
```

Comment:
What if this request takes a long time to load? Try implement a loading state to handle this! Showing the user that the page is loading would be better feedback than nothing - especially because our render servers often need a bit of time to spin up which might lead to a long wait period. We should account for this with some kind of loading message